### PR TITLE
Revert "Update the game over message"

### PR DESCRIPTION
**Pull request by zy4nn**
Update the game over message so that the user can see!

### CODE REVIEW

**Comment by zy4nn** [comment]
Learning!

**Review by zy4nn — COMMENTED** [review]
This is a Comment!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
+<h2 hidden>Game over</h2>
```
Comment (zy4nn) [inline index.html]:
```suggestion
<h2 hidden>Game over!!</h2>
```

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
+<h2 hidden>Game over!!</h2>
```
Comment (s4hill) [inline index.html:50]:
```suggestion
<h2 hidden>Game over!!</h2>
<h2 hidden> This is so cool </h2/
```

Comment (zy4nn) [inline index.html:50]:
```suggestion
<h2 hidden>Game over!!</h2>
<h2 hidden> This is so cool !! </h2>
```

**Pull request merged**